test/hotspot/jtreg/vmTestbase/nsk/jdwp/ThreadReference/OwnedMonitors/ownmonitors001.java

```diff
@@ -98,7 +98,10 @@ public class ownmonitors001 {
      * Start test from command line.
      */
     public static void main (String argv[]) {
-        System.exit(run(argv,System.out) + JCK_STATUS_BASE);
+        int result = run(argv,System.out);
```

Comment:
```suggestion
        int result = run(argv, System.out);
```

Reply:
Thanks, I update the indentation in all places.